modules/smtp/site.js

```diff
@@ -168,6 +168,20 @@ var reset_smtp_form = function() {
     save_compose_state(true);
 };
 
+var get_smtp_profile = function(profile_value) {
+    if (typeof profile_value === "undefined" || profile_value == "0" || profile_value == "") {
+        Hm_Notices.show(['ERRPlease create a profile for saving sent messages option']);
+    }
+    else {
```

Comment:
During the tests, I noticed that this code prevented to see some messages as "Message sent" when the message was sent successfully. 
Even if you remove this condition, the code works because there is a similar condition in the Handler.